intellihide.js

```diff
@@ -23,7 +23,8 @@ const OverlapStatus = {
 const IntellihideMode = {
     ALL_WINDOWS: 0,
     FOCUS_APPLICATION_WINDOWS: 1,
-    MAXIMIZED_WINDOWS : 2
+    MAXIMIZED_WINDOWS: 2,
+    ALWAYS_ON_TOP: 3
```

Comment:
nit: to avoid further extra-diff

```suggestion
    ALWAYS_ON_TOP: 3,
```